ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -565,7 +566,8 @@ data TraceOpenEvent blk =
   | OpenedLgrDB
   deriving (Generic, Eq, Show)
 
--- | Information about the new tip of the current chain.
+-- | Information on having switched our selection to a chain with a
```

Comment:
This data type is also used when merely adding a block to our selection. So I think "switch" is a potential misnomer here.

Reply:
Right, will change to "change" :+1: